### PR TITLE
Added config files to msdk packages

### PR DESCRIPTION
**Pull request by akharche**
Added files to resolve dependencies while installing rpm/deb packages

### CODE REVIEW

File: conf_linux_public.py [hunk 1]
```diff
@@ -292,6 +292,19 @@ def check_lib_size(threshold_size, lib_path):
            stage=stage.INSTALL,
            callfunc=(check_lib_size, [FASTBOOT_LIB_MAX_SIZE, MEDIA_SDK_BUILD_DIR / '__bin/release/libmfxhw64-fastboot.so.{ENV[API_VERSION]}'], {}))
 
+# Create configuration files
+system_config = {
+    f'{options["INSTALL_DIR"]}/intel-mediasdk.conf': f'{MSDK_LIB_INSTALL_DIRS["rpm"]}/lib64',
+    f'{options["INSTALL_DIR"]}/intel-mdf.conf': '/opt/intel/msdk_driver/lib64',
+    f'{options["INSTALL_DIR"]}/intel-mediasdk.sh': '# add libva driver path/name exporting for intel media solution\n'
```
Comment (azhogov) [inline conf_linux_public.py]: should be a part of driver build

File: infrastructure_version.py [hunk 2]
```diff
@@ -19,5 +19,5 @@
 # SOFTWARE.
 
 
-OPEN_SOURCE = {'branch': 'master', 'commit_id': '6fceb7b4d7c4f87dddeaca539f1a722c4cb0b623'}
+OPEN_SOURCE = {'branch': 'add_create_file', 'commit_id': 'b3878da306db3850c7f944a47b8a18c8e4c5623c'}
```
Comment (azhogov) [inline infrastructure_version.py]: change to master

File: conf_linux_public.py [hunk 3]
```diff
@@ -292,6 +292,21 @@ def check_lib_size(threshold_size, lib_path):
            stage=stage.INSTALL,
            callfunc=(check_lib_size, [FASTBOOT_LIB_MAX_SIZE, MEDIA_SDK_BUILD_DIR / '__bin/release/libmfxhw64-fastboot.so.{ENV[API_VERSION]}'], {}))
 
+# Create configuration files
```
Comment (azhogov) [inline conf_linux_public.py]: add TODO: Should be a part of Cmake config

**Comment by azhogov** [comment]
CI failed. Please fix

File: infrastructure_version.py (hunk 2 above)
Comment (akharche) [reply infrastructure_version.py]: Now, I can not change to master, because it depends on not merged commit. That is why CI fails

**Pull request merged**